CoffeeChat/Cells/HomeTabOptionCollectionViewCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  HomeTabOptionCollectionViewCell.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/1/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class HomeTabOptionCollectionViewCell: UICollectionViewCell {
+
+    private let tabLabel = UILabel()
+    private var activeCellColor: UIColor = .darkGreen
+    private var inactiveCellColor: UIColor = .inactiveGreen
```

Comment:
nitpick but can we the var names to be `...TextColor`? just a little more accurate description wise